### DISCUSSION TITLE
Add Paper 'Multi-Garment Net: Learning to Dress 3D People from Images'

### DISCUSSION THREAD

**Discussion by Qingcsai** [3DFaceBody/awesome-3dbody-papers#12] [Ideas]
Hope you could add the paper 'Multi-Garment Net: Learning to Dress 3D People from Images'.
It's a ICCV 2019 paper.
Project link: https://virtualhumans.mpi-inf.mpg.de/mgn/
Paper link: http://virtualhumans.mpi-inf.mpg.de/papers/bhatnagar2019mgn/bhatnagar2019mgn.pdf

**Comment by VVingerfly** [comment]
Hi, @Qingcsai 

Thanks for your recommendation, the paper was added now. :)

B.R.
